src/config/promptUser.ts

```diff
@@ -0,0 +1,21 @@
+import { createInterface } from 'node:readline';
+
+export default function promptUser(message: string): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const rl = createInterface({
+      input: process.stdin,
+      output: process.stdout,
```

Comment:
If we allow input and output to be passed in, we should be able to use real streams in our tests and avoid the mocking. This will make the tests better.